data/ships.txt

```diff
@@ -1480,6 +1480,46 @@ ship "Gunboat"
 
 
 
+ship "Harvester"
+	sprite "ship/mining drone"
+	attributes
+		category "Drone"
+		"cost" 100000
+		"shields" 200
+		"hull" 500
+		"mass" 22
+		"drag" .6
+		"heat dissipation" .92
+		"cargo space" 15
+		"outfit space" 56
+		"weapon capacity" 6
+		"engine capacity" 28
+		"automaton" 1
+		weapon
+			"blast radius" 5
+			"shield damage" 50
+			"hull damage" 25
+			"hit force" 75
+	outfits
+		"Plasma Drill" 2
+
+		"nGVF-AA Fuel Cell"
+		"Supercapacitor"
+
+		"X1200 Ion Steering"
+		"X1700 Ion Thruster"
+
+	gun 15.5 -39
+	gun -15.5 -39
+	engine 6.5 37.5
+	engine -6.5 37.5
+	explode "tiny explosion" 10
+	explode "small explosion" 10
+	description "The recent asteroid mining boom prompted companies to expand their horizons, and start sending out larger fleets of vessels outfitted for mining. Unfortunately, paying to retrofit cargo ships, as well as crew them is expensive, as there are very few ships that are able to effectively mine."
+	description "	These issues prompted shipyards to start accommodating for miners, but it wasn't enough. So, after petitioning the Republic to allow civilian drone use, the large corporations finally got the Republic parliament to give them rights to use drones, as long as they weren't equipped with any offensive armament."
```

Comment:
Description of the other ship says that it's built by Syndicated Shipyards, but this makes it sound like multiple shipyards either produce the Harvester or produce similar drones.